docs/pbc/all/price-management/prices-feature-overview/prices-feature-overview.md

```diff
@@ -19,35 +19,59 @@ redirect_from:
 
-If it still can’t find a price, then it throws an exception. Basically, this doesn't happen if the products have been exported and are up to date.
 
-The following diagram summarizes the logic for retrieving the price for a product:
-![Price retrieval logic](https://spryker.s3.eu-central-1.amazonaws.com/docs/Features/Price/Price+Functionality/price_retrieval_logic.png)
+## Product types and price inheritance
+
+Back Office users can set prices for both [abstract products and product variants](/docs/scos/user/features/{{site.version}}/product-feature-overview/product-feature-overview.html#abstract-products-and-product-variants). When an abstract product has multiple product variants, you can set a price for the abstract product and different prices for each product variant.
+
+On the Storefront, when customers browse catalog and search pages, they see abstract product prices.
+
+![Abstract product prices in catalog](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/price-management/prices-feature-overview/prices-feature-overview.md/abstract-product-prices-in-catalog.png)
+
+When a customer opens a Product Details page, they still see an abstract product price.
+
+![Abstract product price on the Product Details page](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/price-management/prices-feature-overview/prices-feature-overview.md/abstract-product-prices-on-pdp.png)
+
+After selecting a product variant, they see the variant's price.
+
+<iframe width="960" height="720" src="https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/price-management/prices-feature-overview/prices-feature-overview.md/prices-of-abstract-products-and-pruduct-variants.mp4" frameborder="0" allowfullscreen></iframe>
 
-## Price calculation
+In some cases, you may want to set the same price for all the product variants. Then, you set the price for the abstract product and don't set any for the variants. When the variants don't have prices, they inherit the price of their abstract product.
 
-The concerns for the product price calculation are the following:
+Similarly, when there is one product variant, it makes sense to set the price just for the abstract product.
 
-* Retrieve the valid price for the product.
-* Calculate the amount of tax.          
-* Price for the options that were selected for the product (for example, frame or fabric).
+In the last two cases described, since product variants don't have prices, customers see the abstract product price on all the Storefront pages.
+
+## Prices in database
+
+Prices are stored as an integer, in the smallest unit of a currency. For example, for Euro prices are stored in cents.
+
+Each price is assigned to a price type, like gross or net price. For a price type, there can be *one* to *n* product prices defined. Price type entity is used to differentiate between use cases. For example, you can have DEFAULT and ORIGINAL types to use for sale pricing.
```

Comment:
Change gross and net price to DEFAULT and ORIGINAL since gross and net price are price modes and not price types